### PR TITLE
FISH-6460 Update release notes and security fix list

### PR DESCRIPTION
**Pull request by kalinchan**

### CODE REVIEW

**Comment by JamesHillyard** [comment]
Do we want a note about this like we did with the [5.31.0 release](https://docs.payara.fish/enterprise/docs/Release%20Notes/Release%20Notes%205.31.0.html )? CC @Pandrex247

**Review by Pandrex247 — CHANGES_REQUESTED** [review]
You may as well update the security fix list for Community while you're at it.

**Comment by Pandrex247** [comment]
> Do we want a note about this like we did with the [5.31.0 release](https://docs.payara.fish/enterprise/docs/Release%20Notes/Release%20Notes%205.31.0.html)? CC @Pandrex247

Wouldn't go amiss IMO but I'm not the documentation master and not up-to-date with how we're styling things now during Documentation 1.5 / 2.0

**Comment by JamesHillyard** [comment]
> > Do we want a note about this like we did with the [5.31.0 release](https://docs.payara.fish/enterprise/docs/Release%20Notes/Release%20Notes%205.31.0.html)? CC @Pandrex247
> 
> Wouldn't go amiss IMO but I'm not the documentation master and not up-to-date with how we're styling things now during Documentation 1.5 / 2.0

Note added

File: community/docs/modules/ROOT/pages/Security/Security Fix List.adoc [hunk 1]
```diff
@@ -7,6 +7,8 @@ The following is a list of tracked _**C**ommon **V**ulnerabilities and **E**xpos
 |=======================================================================
 |ID |CVSS v3.0 Base Score |Status |Summary |Release |Pull Requests |Observations
 
+|https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-37422[CVE-2022-37422] | N/A | FIXED | Vulnerability exploit using ROOT context root deployments | 5.2022.2 | https://github.com/payara/Payara/pull/5886[#5886] | Recommended to immediately upgrade to this release if any of your applications is deployed on the / context root on Payara Server or Payara Micro
```
Comment (JamesHillyard) [inline community/docs/modules/ROOT/pages/Security/Security Fix List.adoc:10]:
```suggestion
|https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-37422[CVE-2022-37422] | N/A | FIXED | Vulnerability exploit using ROOT context root deployments | 5.2022.3 | https://github.com/payara/Payara/pull/5886[#5886] | Recommended to immediately upgrade to this release if any of your applications is deployed on the / context root on Payara Server or Payara Micro
```

**Review by JamesHillyard — APPROVED** [review]

**Review by AlanRoth — APPROVED** [review]

**Comment by JamesHillyard** [comment]
@Pandrex247 could you re-review this so it can be merged please?

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**